File: Java/Tree ADT/BinaryTree/src/BTNode.java
class BTNode
{
  private String data;
  private BTNode left, right;

  public BTNode(String s)
  {
    data = s;
    left = null;
    right = null;
  }

  public String getData()
  {
    return data;
  }

  public void setData(String s)
  {
    data = s;
  }

  public BTNode getLeft()
  {
    return left;
  }

  public void setLeft(BTNode n)
  {
    left = n;
  }

  public BTNode getRight()
  {
    return right;
  }

  public void setRight(BTNode n)
  {
    right = n;
  }

}

File: Python/Library Repo/BTNode.py
class BTNode():

  def __init__(self, val):
    self.data  = val
    self.left  = None
    self.right = None
  
  def getData(self):
    return self.data

  def getLeft(self):
    return self.left

  def getRight(self):
    return self.right

  def setLeft(self, newVal):
    self.left = BTNode(newVal)

  def setRight(self, newVal):
    self.right = BTNode(newVal)

  def isLeaf(self):
    return self.left == None and self.right == None